### DISCUSSION TITLE
Cannot build special configurations - Linear advance

### DISCUSSION THREAD

**Discussion by curather** [mriscoc/Ender3V2S1#384] [General]
The linear advance is using configuration h version 02010200 while the ender 3v2s1 is using 02010100.  This gives a mismatch when trying to compile using the config files from there.  In the version.h file, it says

`#define DETAILED_BUILD_VERSION SHORT_BUILD_VERSION " Ender3V2-422-BLTUBL-LA, based on bugfix-2.1.x"`

Will we get a updated release soon so we can compile based off the bugfix-2.1.x?

**Comment by mriscoc** [comment]
https://github.com/mriscoc/Ender3V2S1/discussions/365
